tests/RobotFramework/tests/aws/aws_coonect_with_keys.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Documentation       Verify that thin-edge.io can successfully connect to AWS IoT Core
+...                 Test assumes that AWS credentials are added to the .env file
+...                 AWS_URL=
+...                 AWS_ACCESS_KEY=
+...                 AWS_SECRET_KEY=
+...                 AWS_REGION=
+...    
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO    adapter=ssh
+
+
+Suite Setup         Custom Setup
+Test Teardown      Custom Teardown
+
+Test Tags           theme:aws    test:on_demand
+
+*** Variables ***
+${POLICY_NAME}=       thinedge.io
+${CERT_PATH}=         /etc/tedge/device-certs/tedge-certificate.pem
+${ROOT_CA_PATH}=      /etc/tedge/device-certs/tedge-certificate.pem
+
+
+
+*** Test Cases ***
+
+Create AWS IoT Policy and Thing  
+    #Check addapter and run only if adapter is ssh
+    Run Command If SSH Adapter
```

Comment:
Why such a restriction?

Reply:
One of the differences of the "Setup" keyword wtih different adapter used is that in case of adapter=Docker (adapter: None is default, which is Docker) Setup is bootstrapping and making a fresh instalation, creating certificate etc. In case of adapter=ssh it is not